src/components/Counter.js

```diff
@@ -1,59 +1,58 @@
 import React, { Component } from "react";
-import { connect } from 'react-redux';
-import { increment, decrement } from '../actions';
+import { connect } from "react-redux";
+import { increment, decrement } from "../actions";
 
 class Counter extends Component {
-    incrementIfOdd = () => {
-        // Stretch Problem: Implement an increment function that
-        // only increments if the counter value is odd
-    };
+  incrementIfOdd = () => {
+    // Stretch Problem: Implement an increment function that
+    // only increments if the counter value is odd
+    if (this.props.count % 2 !== 0) {
```

Comment:
What other ways can you get truthy for odd?

Reply:
if (this.props.count % 2 === 1) or (this.props.count % 3 === 0)   should both get truthy for odd, and for the mvp completed twice i meant to say stretch on the second commit